docs/current/quickstart/730264-quickstart-publish.mdx

```diff
@@ -13,9 +13,9 @@ import QuickstartDoc from "@site/src/components/molecules/quickstartDoc.js"
 import Embed from "@site/src/components/atoms/embed.js"
 
 export const ids = {
-    Go: "SYJ885fBPOy",
-    Node: "P8mz6yjC2lM",
-    Python: "Vz6OJ6gIDaP"
+    Go: "",
```

Comment:
@marcosnils I am not able to get the playground to execute this embed consistently. I sometimes see a "gateway timeout" or "network error" both when using the snippet and when directly entering the code into the web playground: https://play.dagger.cloud/playground/BnZUZFuzq9b

Reply:
@marcosnils the CI also shows this error for the same file: `Error: docs/current/quickstart/snippets/publish/main.go:46:67: G404: Use of weak random number generator (math/rand instead of crypto/rand) (gosec)
		Publish(ctx, fmt.Sprintf("ttl.sh/hello-dagger-%.0f", math.Floor(rand.Float64()*10000000)))`. Can you please suggest a fix?

Reply:
hey Vikram! if you add `//nosec` at the end of the line it should ignore that error.

Reply:
> I am not able to get the playground to execute this embed consistently. I sometimes see a "gateway timeout" or "network error" both when using the snippet and when directly entering the code into the web playground: https://play.dagger.cloud/playground/BnZUZFuzq9b

I'll check it out tomorrow :pray: